### PR TITLE
Add example script to print out all updates

### PR DESCRIPTION
**Pull request by expectocode**

### CODE REVIEW

File: telethon_examples/print_updates.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+from telethon import TelegramClient
+from getpass import getpass
+from os import environ
+# environ is used to get API information from environment variables
+# You could also use a config file, pass them as arguments, 
+# or even hardcode them (not recommended)
+
+def main():
+    session_name = 'session'
```
Comment (Lonami) [inline telethon_examples/print_updates.py]: `environ.get('TG_SESSION', 'session')`

File: telethon_examples/print_updates.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    user_phone = environ['TG_PHONE'],
+    client = TelegramClient(session_name,
+            int(environ['TG_API_ID']), 
+                environ['TG_API_HASH'],
+                proxy=None,
+                update_workers=4)
+
+    print('INFO: Connecting to Telegram Servers...', end='', flush=True)
```
Comment (Lonami) [inline telethon_examples/print_updates.py]: `logging.info(...)`

File: telethon_examples/print_updates.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    print('Done!')
+
+    if not client.is_user_authorized():
+        print('INFO: Unauthorized user')
+        client.send_code_request(user_phone)
+        code_ok = False
+        while not code_ok:
+            code = input('Enter the auth code: ')
+            try:
+                code_ok = client.sign_in(user_phone, code)
```
Comment (Lonami) [inline telethon_examples/print_updates.py]: FYI, `.sign_in` returns the user, not `True`.

File: telethon_examples/print_updates.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            except SessionPasswordNeededError:
+                pw = getpass('Two step verification enabled. Please enter your password: ')
+                client.sign_in(password=pw)
```
Comment (Lonami) [inline telethon_examples/print_updates.py]: Not setting `code_ok` here?

File: telethon_examples/print_updates.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    print('INFO: Client initialized succesfully!')
+
+    client.add_update_handler(update_handler)
+    input('Press Enter to stop this!\n')
```
Comment (Lonami) [inline telethon_examples/print_updates.py]: Have you tried the code? I would swear this just exits immediatly, without calling `.disconnect()` which is a bad practice.

**Comment by Lonami** [comment]
Works for me. Thanks.

**Comment by Lonami** [comment]
Now go fix "auto replier".

**Comment by andreif** [comment]
@Lonami There is unresolved reference in the code... and no pep8...

**Pull request merged**